codec-http/src/main/java/io/netty/handler/codec/http/multipart/DefaultHttpDataFactory.java

```diff
@@ -111,6 +116,14 @@ public DefaultHttpDataFactory(long minSize, Charset charset) {
         this.charset = charset;
     }
 
+    public void setBaseDir(String baseDir) {
+        this.baseDir = baseDir;
+    }
+
+    public void setDeleteOnExit(boolean deleteOnExit) {
```

Comment:
@rmannibucau can you please add java doc for both of the methods ?

Reply:
done, thanks for the catch up